packages/workflow/src/records/handler/create.ts

```diff
@@ -360,6 +364,24 @@ export default async function createRecordHandler(
       await indexBundle(rejectedOrWaitingValidationRecord, token)
       await auditEvent('sent-for-updates', record, token)
     }
+  } else {
+    /*
+     * Notify country configuration about the event so that countries can hook into actions like "sent-for-approval"
+     */
+    const recordSpecificToken = await getRecordSpecificToken(
+      token,
+      request.headers,
+      getComposition(record).id
+    )
+    await notifyForAction({
+      event,
+      action: eventAction,
+      record,
+      headers: {
+        ...request.headers,
+        authorization: `Bearer ${recordSpecificToken.access_token}`
+      }
+    })
```

Comment:
Are we going with the assumption that every country would want this communication to be asynchronous?

Reply:
👍Good point.. I guess the `await` gives a chance for that? You can then handle it asynchronously in country-config if you like 🤔

Reply:
Ah yeah, get what you mean now. Yes, currently that is the assumption. I think that's fine for starters, and if we get a need for discontinuing the flow similarly like event registration can reject or confirm, we can re-think this